Realm/RLMSyncUser.mm

```diff
@@ -317,7 +317,8 @@ - (void)changePassword:(NSString *)newPassword forUserID:(NSString *)userID comp
                                      server:self.authenticationServer
                                        JSON:@{kRLMSyncTokenKey: self._refreshToken,
                                               kRLMSyncUserIDKey: userID,
-                                              kRLMSyncPasswordKey: newPassword}
+                                              kRLMSyncDataKey: @{ @"new_password": newPassword }
```

Comment:
Should `"new_password"` be pulled out into a constant like all of the other keys?